pine-a64-plus/armbian.yaml

```diff
@@ -0,0 +1,93 @@
+
+{{- $image := or .image "debos-debian-bullseye-pine-a64.img" -}}
+
+architecture: "arm64"
+
+
+actions:
+  - action: debootstrap
+    suite: "bullseye"
+    components:
+      - main
+      - contrib
+      - non-free
+    mirror: https://deb.debian.org/debian
+    variant: minbase
+
+  - action: apt
+    description: Install some packages
+    packages: 
+        - sudo
+        - openssh-server
+        - isc-dhcp-client
+        - iproute2
+        - ifupdown
+
+# Armbian-specific (kernel, bootloader and settings)
+
+  - action: overlay
+    source: overlays/armbian
+
+  - action: apt
+    description: Kernel, U-Boot, DeviceTree install for Pine A64 (armbian)
+    packages: 
+        - armbian-firmware
+        - linux-u-boot-pine64-current
+        - linux-image-current-sunxi64
+        - linux-dtb-current-sunxi64
+        - sunxi-tools 
+        - armbian-bsp-cli-pinebook-a64
+        - init
+        - initramfs-tools
+        
+  - action: run
+    description: Generate initramfs
+    chroot: true
+    command: update-initramfs -k all -c 
+
+# End of Armbian-specific
+
+  - action: run
+    chroot: true
+    command: echo debos-debian-pine-a64 > /etc/hostname
+
+  - action: run
+    description: Set up user
+    chroot: true
+    script: scripts/setup-user.sh
+
+  - action: run
+    description: Network setup
+    chroot: true
+    script: scripts/setup-network.sh
+
+  - action: image-partition
+    imagename: {{ $image }}
+    imagesize: 1GB
+    partitiontype: msdos
+    mountpoints:
+      - mountpoint: /
+        partition: root
+    partitions:
+      - name: root
+        fs: ext4
+        start: 1% 
```

Comment:
as far as I remember the gap for the bootloader should have the fixed size. Start in percentage is dangerous and non-optimal.